src/blog/2024/10/announcement-mqtt-broker.md

```diff
@@ -0,0 +1,76 @@
+---
+title: "MQTT Service Now Available on FlowFuse"
+subtitle: We are thrilled to announce a significant milestone for FlowFuse, we now offer our very own MQTT service, built-in and ready to use with your node-RED applications.
+description: We are thrilled to announce a significant milestone for FlowFuse, we now offer our very own MQTT service, built-in and ready to use with your node-RED applications.
+date: 2024-10-30
+authors: ["joe-pavitt"]
+image: /blog/2024/10/images/dashboard-1-18-0-blog-tile.png
+tags:
+   - posts
+   - news
+   - flowfuse
+   - node-red
+---
+
+You may have seen in our [recent product update](/blog/2024/10/flowfuse-release-2-10) that we have release our very own MQTT service, built-in and ready to use alongside your Node-RED applications. We are always engaging with users and prospective customers and this has consistently been a request, and so we are delighted to announce that this is now live on FlowFuse Cloud for our Team and Enterprise teams.
```

Comment:
"we have release our very own MQTT service"

Apart from "release" should be "released", whould this not be that we have "we have added" instead?

Reply:
Sensible, feel free to make the change when you're adding in the other paragraph

Reply:
```suggestion
You may have seen in our [recent product update](/blog/2024/10/flowfuse-release-2-10) that we have added our very own MQTT service, built-in and ready to use alongside your Node-RED applications. We are always engaging with users and prospective customers and this has been a highly requested feature, and so we are delighted to announce that this is now live on FlowFuse Cloud for our Team and Enterprise teams.
```

This is my take Joe.